setup/src/Magento/Setup/Module/Di/Code/Generator/InterceptionConfigurationBuilder.php

```diff
@@ -48,25 +51,33 @@ class InterceptionConfigurationBuilder
      */
     private $interceptableValidator;
 
+    /**
+     * @var ConfigInterface
+     */
+    private $omConfig;
+
     /**
      * @param InterceptionConfig $interceptionConfig
      * @param PluginList $pluginList
      * @param Type $typeReader
      * @param Manager $cacheManager
      * @param InterceptableValidator $interceptableValidator
+     * @param ConfigInterface|null $omConfig
      */
     public function __construct(
         InterceptionConfig $interceptionConfig,
         PluginList $pluginList,
         Type $typeReader,
         Manager $cacheManager,
-        InterceptableValidator $interceptableValidator
+        InterceptableValidator $interceptableValidator,
+        ?ConfigInterface $omConfig = null
     ) {
-        $this->interceptionConfig = $interceptionConfig;
-        $this->pluginList = $pluginList;
-        $this->typeReader = $typeReader;
-        $this->cacheManager = $cacheManager;
+        $this->interceptionConfig     = $interceptionConfig;
+        $this->pluginList             = $pluginList;
+        $this->typeReader             = $typeReader;
+        $this->cacheManager           = $cacheManager;
```

Comment:
@GrassH Could you please rollback formatting

Reply:
@Den4ik rolled back, please review again.